source/Generic/GamesSizeCalculator/Steam/SteamSizeCalculator.cs

```diff
@@ -9,84 +10,160 @@
 
 namespace GamesSizeCalculator.SteamSizeCalculation
 {
-    public class SteamSizeCalculator
+    public class SteamSizeCalculator : IOnlineSizeCalculator
     {
         private ILogger logger = LogManager.GetLogger();
```

Comment:
This one can be set as static

Reply:
Unclear what you mean here

Reply:
private static ILogger logger = LogManager.GetLogger();

But it can be kept this way too